admin-dev/themes/new-theme/scss/components/layout/_main_header.scss

```diff
@@ -548,7 +548,8 @@ $header-text-color: #4e6167;
       border: 1px solid #899EA5;
       border-radius: 1px;
       color: #576C72;
-      max-width: 130px;
+      min-width: 130px;
+      width: fit-content;
```

Comment:
:warning: fit-content is not supported for IE or Edge

Reply:
Woot, thanks for advice. Forget to check these browsers :sweat_smile: